src/modules/help.ts

```diff
@@ -91,10 +92,21 @@ export class HelpModule extends Module {
 
 		const cmd = this.client.commandManager.getByTrigger(cmdTrigger);
 		if (!cmd || !cmd.description) {
-			await sendWithMessageOwnership(
-				msg,
-				`:x: Command "${cmdTrigger}" not found`,
-			);
+			const shortcut = await Shortcut.findOne(cmdTrigger);
```

Comment:
It'd be nicer if this was refactored so there aren't two codepaths that produce similar embeds. (this and the one at [L113](https://github.com/typescript-community/community-bot/blob/ffeddcc82d76a837cd5be05a88be3a87d33deb4d/src/modules/help.ts#L113))